docs/dev/Creating UI5 Web Components Packages.md

```diff
@@ -107,42 +107,42 @@ That's it!
 
 ## Understanding the project structure
```

Comment:
I'd move the whole of the second part (Understanding the project structure) into a separate .md file/topic.